sdks/python/apache_beam/runners/worker/statesampler.py

```diff
@@ -79,3 +109,18 @@ def commit_counters(self):
     for state in self._states_by_name.values():
       state_msecs = int(1e-6 * state.nsecs)
       state.counter.update(state_msecs - state.counter.value())
+
+
+def simple_tracker():
```

Comment:
If this is just used for tests, put it in a test file. I would rather force the callers to provide a prefix and counter factory manually.